policies/model/src/main/java/org/eclipse/ditto/policies/model/signals/events/PolicyActionEvent.java

```diff
@@ -28,5 +28,5 @@ public interface PolicyActionEvent<T extends PolicyActionEvent<T>> extends Polic
      * @param otherPolicyActionEvents the collection of policy action events.
      * @return the aggregated event.
      */
-    PolicyEvent<?> aggregateWith(Collection<PolicyActionEvent<?>> otherPolicyActionEvents);
+    PolicyEvent<?> aggregateWith(Collection<PolicyActionEvent> otherPolicyActionEvents);
```

Comment:
I didn't quite got the idea of removing the wildcard parameter type here. What is the reason for that?

Reply:
Yes, I "had to" do this as I had otherwise compile errors all over the place when adding the `CompletionStage` overloaded methods to e.g. `Result` and `ResultVisitor`.
So not really "nicer" that way, however I did not see another way.